docs/monitoring/metrics.md

```diff
@@ -896,6 +896,28 @@ Thus, in order to infer the metric identifier:
   </tbody>
 </table>
 
+#### Connector:
```

Comment:
Let's name this heading "Connectors", and add a new sub-heading specifically for kafka.